### PR TITLE
Update to 0.9.0-dev.950+93c6e31cf

### PR DESCRIPTION
**Pull request by leroycep**

### CODE REVIEW

File: lmdb.zig [hunk 1]
```diff
@@ -123,8 +123,8 @@ pub const Environment = packed struct {
             try call(c.mdb_env_copy2, .{ self.inner, backup_path.ptr, flags.into() });
         }
     }
-    pub inline fn pipeTo(self: Self, fd: os.fd_t, flags: CopyFlags) !void {
-        try call(c.mdb_env_copyfd2, .{ self.inner, fd, flags.into() });
+    pub inline fn pipeTo(self: Self, fdHandle: os.fd_t, flags: CopyFlags) !void {
```
Comment (lithdew) [inline lmdb.zig]: Keep all variable names except for struct and function names snake-cased. Also, does a compile error happen if the parameter name is unchanged?

```suggestion
    pub inline fn pipeTo(self: Self, fd_handle: os.fd_t, flags: CopyFlags) !void {
```

File: lmdb.zig [hunk 2]
```diff
@@ -123,8 +123,8 @@ pub const Environment = packed struct {
...
+        try call(c.mdb_env_copyfd2, .{ self.inner, fdHandle, flags.into() });
```
Comment (lithdew) [inline lmdb.zig]:
```suggestion
        try call(c.mdb_env_copyfd2, .{ self.inner, fd_handle, flags.into() });
```

**Review by lithdew — CHANGES_REQUESTED** [review]
Wow, I'm glad the new Zig update managed to catch that small typo :sweat_smile:. Thanks as always for helping keep this library up to date. Just one small little nit.

File: lmdb.zig (hunk 1 above)
Comment (leroycep) [reply lmdb.zig]: Yes, it was complaining about the parameter shadowing the function on line 240.

**Comment by lithdew** [comment]
LGTM!

**Pull request merged**